cmd/boost/provider_cmd.go

```diff
@@ -174,7 +187,8 @@ var storageAskCmd = &cli.Command{
 		}
 
 		if err := doRpc(ctx, s, &askRequest, &resp); err != nil {
-			return fmt.Errorf("send ask request rpc: %w", err)
+			augmentedError := fmt.Errorf("send ask request rpc: %w", err)
+			return cmd.PrintError(augmentedError, outputInJson)
 		}
 
 		ask := resp.Ask.Ask
```

Comment:
Why do we use the `afmt` special command here and not everywhere else?
Why don't we support `json` output for this command as well?